Automattic/pocket-casts-android
# Tasker plugin new action

## Description
- When using the **Pocket Casts Control Playback** action in Tasker there are now 3 new options for the Playback Command: **Set Playback Speed**, **Set Trim Silence Mode** and **Set Volume Boost**
- Added icons to Tasker action input fields
- Code cleanup was made so that it's easier to add new fields to Tasker actions (you don't need to edit code in so many places as before so it's less error prone)

Partly Fixes https://github.com/Automattic/pocket-casts-android/issues/471 (the part about adding icons)

**Important note**: Review is needed because although the above effects are correctly applied to playback, you can't see their effects on the UI, eg when you change playback speed via Tasker the UI still shows the old value that was set via the UI. Since I'm not that familiar with the app's architecture as a whole I would appreciate a little help with finding out why this is happening 🙂

## Testing Instructions
- Open Tasker
- Add **Pocket Casts Control Playback** action
- Set **Playback Command** to **Skip Forward**
- Select of the new **Set Playback Speed**, **Set Trim Silence Mode** or **Set Volume Boost** commands
- Select the appropriate value for that command
- Press Ok
- Start playing a podcast in Pocket Casts
- Run the Tasker action and check that the effects are applied to playback

## Checklist
 
#### I have tested any UI changes...
